docs/content/configuration/csi_ug_config_create_replication.md

```diff
@@ -0,0 +1,45 @@
+# Creating a VolumeReplication
+
+Create a VolumeReplication YAML file to replicate a specific PersistentVolumeClaim (PVC).
+
+VolumeReplicationClass needs to be present before a VolumeReplication can be created. For more information, see [Creating a VolumeReplicationClass](csi_ug_config_create_vol_replicationclass.md).
+
+**Note:** Replication is referred to as the more generic volume replication within this documentation set. Spectrum Virtualize products refer to this feature as the remote copy function.
+
+When replicating a volume, be sure to follow all of the replication configurations, found in [Compatibility and requirements](../installation/csi_ug_requirements.md) before volume replication.
+
+1.  Replicate a specific PersistentVolumeClaim (PVC) using the `demo-volumereplication.yaml`.
+
+    For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+
+    ```
+    apiVersion: replication.storage.openshift.io/v1alpha1
+    kind: VolumeReplication
+    metadata:
+      name: demo-volumereplication
+      namespace: default
+    spec:
+      volumeReplicationClass: demo-volumereplicationclass
+      replicationState: primary
+      dataSource:
+        kind: PersistentVolumeClaim
+        name: demo-pvc-file-system # should be in same namespace as VolumeReplication
```

Comment:
why are we using lowercase for the comment here?

Reply:
I don't know. check the latest yaml file and if it's ok there. i still need to go through and update all of the yamls in my documentation.

Reply:
updated all yaml files in my docs. please review again

Reply:
I don't see the change

Reply:
I see it, [here](https://github.com/IBM/ibm-block-csi-driver/pull/360/files#diff-3b5a6baab6dd9dbaa701947f25e694714ae057df1f6a9cfd3a5dd9a851c3f64e)